lib/iris/analysis/trajectory.py

```diff
@@ -133,6 +133,74 @@ def __repr__(self):
         return 'Trajectory(%s, sample_count=%s)' % (self.waypoints,
                                                     self.sample_count)
 
+    def _get_interp_points(self):
+        """
+        Translate `self.sampled_points` to the format expected by the
+        interpolator. If the CRS of the sample points does not match the CRS
```

Comment:
Hate to say it, but I think you provide no test for the CRS-transforming behaviour.
I can't see the code for it either, maybe implicit in existing code -- or **is it actually doing this ?**

Reply:
Actually, the problem is that the docstring is now out of date – I dropped CRS transforming early on when it became evident it was too much to try and achieve. Thanks for the heads-up though @pp-mo, I'll update the docstring.